tools/maf_cleanup.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python
+import optparse
+
+### start of parsing parameters
+parser = optparse.OptionParser()
+parser.add_option('-i', '--input-file', action = 'store', dest = 'inputfile', help = "file name of input maf")
+parser.add_option('-o', '--output-file', action= 'store', dest = "outputfile", default=False, help = "file name of output maf")
+parser.add_option('-c', '--gene-id', action= 'store', dest = "geneidfile", default=False, help = "file name of gene id mapping")
```

Comment:
If the `gene-id` file accompanies this code, it's possible to remove this argument and hardcode its location.

Reply:
@averyniceday was suggest to change it as an input.